person/admin.py

```diff
@@ -1,10 +1,5 @@
 from django.contrib import admin
-from .models import Person
+from .models import *
+from django.contrib.auth.admin import UserAdmin
```

Comment:
'django.contrib.auth.admin.UserAdmin' imported but unused

Reply:
i will delete it